<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HidraGui</class>
 <widget class="QMainWindow" name="HidraGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hidra - PET Computação</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="areaEditor" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layoutSourceCodeHolder"/>
       </item>
       <item>
        <widget class="QLabel" name="labelMessages">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Mensagens:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditError">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>61</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableViewMemoryInstructions">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableViewMemoryData">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="areaSimulator" native="true">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>301</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBoxMachine">
         <item>
          <property name="text">
           <string>Neander</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ahmes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ramses</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="areaRegisters">
         <property name="title">
          <string>Registradores</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollAreaRegisters">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="widgetRegisters">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>279</width>
               <height>175</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="spacerRegistersLeft">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" name="layoutRegisters">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>6</number>
                </property>
               </layout>
              </item>
              <item>
               <spacer name="spacerRegistersRight">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="areaFlags">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>3</number>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="layoutFlags">
            <property name="spacing">
             <number>18</number>
            </property>
           </layout>
          </item>
          <item row="0" column="0">
           <spacer name="spacerFlagsLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="spacerFlagsRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="areaInstructions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="title">
          <string>Instruções</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>3</number>
          </property>
          <item>
           <spacer name="spacerInstructionsLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="layoutInstructions">
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
           </layout>
          </item>
          <item>
           <spacer name="spacerInstructionsRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonBuild">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Montar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRun">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="text">
            <string>Rodar</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonStep">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Passo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>areaEditor</zorder>
   <zorder>areaSimulator</zorder>
   <zorder>tableViewMemoryInstructions</zorder>
   <zorder>tableViewMemoryData</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="actionReportProblem"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMemory">
    <property name="title">
     <string>Máquina</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionRun"/>
    <addaction name="actionStep"/>
    <addaction name="separator"/>
    <addaction name="actionResetRegisters"/>
    <addaction name="actionSetBreakpoint"/>
    <addaction name="separator"/>
    <addaction name="actionImportMemory"/>
    <addaction name="actionExportMemory"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Exibir</string>
    </property>
    <addaction name="actionDisplayDataTable"/>
    <addaction name="actionHexadecimalMode"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuMemory"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBuild">
   <property name="text">
    <string>Montar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Passo</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Rodar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Salvar como...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionReportProblem">
   <property name="text">
    <string>Relatar um problema</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Fechar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImportMemory">
   <property name="text">
    <string>Importar memória...</string>
   </property>
  </action>
  <action name="actionExportMemory">
   <property name="text">
    <string>Exportar memória...</string>
   </property>
  </action>
  <action name="actionResetRegisters">
   <property name="text">
    <string>Zerar registradores</string>
   </property>
  </action>
  <action name="actionHexadecimalMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Modo hexadecimal</string>
   </property>
  </action>
  <action name="actionSetBreakpoint">
   <property name="text">
    <string>Inserir/remover breakpoint</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionDisplayDataTable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tabela de dados</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
